docs/content/preview/drivers-orms/csharp/postgres-npgsql.md

```diff
@@ -72,9 +70,8 @@ NpgsqlConnection conn = new NpgsqlConnection(yburl)
 | user | User for connecting to the database | yugabyte
 | password | Password for connecting to the database | yugabyte
 
-#### Using SSL
-
-The .NET Npgsql driver validates certificates differently from other PostgreSQL drivers. When you specify SSL mode `require`, the driver verifies the certificate by default (like the `verify-ca` or `verify-full` modes), and fails for self-signed certificates (like YugabyteDB's). You can override this by specifying "Trust Server Certificate=true", in which case it bypasses walking the certificate chain to validate trust and hence works like other drivers' `require` mode. In this case, the Root-CA certificate is not required to be configured.
+Set up the driver properties to configure the credentials and SSL Certificates for connecting to your cluster.
+The .NET Npgsql driver validates certificates differently from other PostgreSQL drivers. When you specify SSL mode `require`, the driver verifies the certificate by default (like the `verify-ca` or `verify-full` modes), and fails for self-signed certificates (like YugabyteDB's). You can override this by specifying `Trust Server Certificate = true`, in which case it bypasses walking the certificate chain to validate trust, and hence works like other drivers' `require` mode. In this case, the Root-CA certificate is not required to be configured.
```

Comment:
extra line break or new paragraph?

For a local install, do I even use a certificate? Where would I get one? YB Managed uses certificates and normally works with verify-full; is it something about this driver that keeps it from working? ie, require + Trust Server Certificate = true is not the same as verify-full?
Following the Use C# with SSL link, table lists verify-full and verify-ca, but then the note says they aren't supported.